lib/stream.js

```diff
@@ -77,21 +76,13 @@ const getSpawnedResult = async ({stdout, stderr, all}, {encoding, buffer, maxBuf
 			{error, signal: error.signal, timedOut: error.timedOut},
 			getBufferedData(stdout, stdoutPromise),
 			getBufferedData(stderr, stderrPromise),
-			getBufferedData(all, allPromise)
+			getBufferedData(all, allPromise),
 		]);
 	}
 };
 
-const validateInputSync = ({input}) => {
+export const validateInputSync = ({input}) => {
 	if (isStream(input)) {
 		throw new TypeError('The `input` option cannot be a stream in sync mode');
 	}
 };
-
-module.exports = {
```

Comment:
Is there a preference for `export { foo, bar } `  vs individually exporting (`export const foo`)?